tests/niminaction/Chapter9/configurator/configurator.nim

```diff
@@ -47,7 +47,7 @@ proc createLoadProc(typeName: NimIdent, identDefs: seq[NimNode]): NimNode =
         `cfgIdent`.`fieldNameIdent` = `objIdent`[`fieldName`].getStr
     of "int":
       body.add quote do:
-        `cfgIdent`.`fieldNameIdent` = `objIdent`[`fieldName`].getNum().int
```

Comment:
Please don't change this. We need to make sure that whatever is in the book continues to work. That's the whole point.